sdk/formrecognizer/perf-tests/ai-form-recognizer/test/custom.spec.ts

```diff
@@ -4,19 +4,23 @@
 import { PerfOptionDictionary, PerfTest, getEnvVar } from "@azure/test-utils-perf";
 import {
   AzureKeyCredential,
-  BeginRecognizeCustomFormsOptions,
-  CustomFormModel,
-  FormRecognizerClient,
-  FormTrainingClient,
+  DocumentModelDetails,
+  DocumentAnalysisClient,
+  DocumentModelAdministrationClient,
 } from "@azure/ai-form-recognizer";
 import { DefaultAzureCredential, TokenCredential } from "@azure/identity";
+import { v4 as generateUuid } from "uuid";
```

Comment:
could you use the `randomUUID` from core-util instead?
https://github.com/Azure/azure-sdk-for-js/blob/74ff4f639db914436c1666ab617b6c4d98c6ce1f/sdk/core/core-util/src/uuidUtils.ts#L31

Reply:
done.